src/wallet/wallet.cpp

```diff
@@ -1558,7 +1560,12 @@ CBlockIndex* CWallet::ScanForWalletTransactions(CBlockIndex* pindexStart, bool f
                 LogPrintf("Still rescanning. At block %d. Progress=%f\n", pindex->nHeight, GuessVerificationProgress(chainParams.TxData(), pindex));
             }
         }
+        if (pindex && fAbortRescan) {
```

Comment:
It's set at line 1530 above. I was thinking of setting it to false at end but it doesn't really matter as it's reset at top anyway.